nolearn/lasagne/base.py

```diff
@@ -90,8 +101,8 @@ def __init__(
         self,
         layers,
         update=nesterov_momentum,
-        loss=None,
-        objective=Objective,
+        loss=None,  # BBB
```

Comment:
Oh that's just an informal comment to remind me that this line is only here for backward compatibility and can go away in the future.